evm/src/NttManager/ManagerBase.sol

```diff
@@ -372,7 +419,7 @@ abstract contract ManagerBase is
     }
 
     /// @inheritdoc IManagerBase
-    function setThreshold(uint8 threshold) external onlyOwner {
+    function setThreshold(uint8 threshold) external onlyOwner outboundWhenPaused {
```

Comment:
Isn't the threshold of the local chain more concerned around receiving inbound messages? For example a deployment may want to change the threshold on one chain only, and they would only have to pause the outbound transfer of that chain, not every other chain that sends to it?